cockroachdb/values.yaml

```diff
@@ -176,19 +176,60 @@ conf:
   http-port: ""
 
   # CockroachDB's data mount path.
+  # For multi-store configuration, the path for each store is evaluated as:
+  # Store 1: cockroach-data
+  # Store 2: cockroach-data-2
+  # Store N: cockroach-data-N
   path: cockroach-data
 
   # CockroachDB's storage configuration https://www.cockroachlabs.com/docs/v21.1/cockroach-start.html#storage
   # Uses --store flag
   store:
     enabled: false
+    # Number of data stores per node.
+    # For multi-store configuration, set this to a value greater than 1.
+    count: 1
     # Should be empty or 'mem'
     type:
     # Required for type=mem. If type and size is empty - storage.persistentVolume.size is used
     size:
     # Arbitrary strings, separated by colons, specifying disk type or capability
     attrs:
 
+  # CockroachDB's WAL failover configuration:
+  # https://www.cockroachlabs.com/docs/stable/cockroach-start#write-ahead-log-wal-failover
+  # Uses `--wal-failover` flag
+  wal-failover:
+    # The value to be passed to the `--wal-failover` flag.
+    # Possible configurations:
+    # 1. Default: If empty, `--wal-failover` is not passed to cockroach start
```

Comment:
Nit: we can have values in the beginning to make it more readable

Possible configurations : 
1. `` : When empty, the configuration is ignored and  `--wal-failover` is not passed to cockroach start, this is also the default behavior. 
2. `disabled` : Disables WAL failover. 
3. `among-stores` : Enables WAL failover  among multiple stores. Setting `conf.store.count` greater than 1 is a dependency for this to to take effect. 
4. `path=<path-to-side-disk>`: Enables WAL failover on a side disk. A persistent volume should be mounted at the specified path (e.g. `path=/cockroach/cockroach-failover`).

Reply:
Addressed.